dll/win32/msgina/lang/zh-CN.rc

```diff
@@ -175,39 +175,39 @@ BEGIN
     DEFPUSHBUTTON "确定", IDOK, 115, 179, 50, 14
 END
 
-IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 208, 122
+IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 208, 121
```

Comment:
Same comment as in japanese translation regarding the sizes of the shutdown dialogs and its controls inside.